hyperref.dtx

```diff
@@ -1231,6 +1231,9 @@
 %    There are several commands that prints characters in the
 %    printable ASCII area that don't obey the NFSS, so they have
 %    to be redefined here.
+%    UF 29.09.2017: added a mapping for \noboundary, see issue #37
```

Comment:
I think there's some tex markup missing here for typeset comments, but I'll take the PR and fix locally